scripts/utils/verify.sh

```diff
@@ -21,26 +21,25 @@
 #     test/mocks/MockToken.sol:MockToken
 
 # SovereignPool Verify Script
+forge verify-contract \
+    --chain-id 1 \
+    --num-of-optimizations 20000 \
+    --watch \
+    --constructor-args $(cast abi-encode "constructor(address,address,address,address,address,address,bool,bool,uint256,uint256,uint256)" 0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48 0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2 0x502C7C62CeF3D7314597532510573b87663EF85b 0x502C7C62CeF3D7314597532510573b87663EF85b 0x0000000000000000000000000000000000000000 0x0000000000000000000000000000000000000000 false false 0 0 0) \
+    --etherscan-api-key 55UPD4PWFW6GNVHDAI54RGUD7D6NWEYMV9 \
```

Comment:
I don't think we should expose our etherscan api key, is there a way we access it here from .env file?

Reply:
done, I can just add the API key when we are verifying, we don't use this script frequently